.eslintrc.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     ReactDOM: true,
   },
 
-  rules: { 'react/jsx-uses-vars': 'error' },
+  rules: { 'react/jsx-uses-vars': 'error', 'react/jsx-uses-react': 'error' },
```

Comment:
Needed since I added explicit React import which is required to be able to use component in `webpack` from node_modules without script tags